### DISCUSSION TITLE
Quarkus 3.0.0.Alpha3 released - Third iteration of our Jakarta EE 10 stream

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#30676] [Quarkus Blog/Website]
# Quarkus 3.0.0.Alpha3 released - Third iteration of our Jakarta EE 10 stream

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-3-0-0-alpha3-released/

**Comment by meggarr** [comment]
Do we have a Candidate Release before the 3.0.0.Final ?

**Reply by gsmet** [reply]
Sure. There will be a couple more alphas, maybe some betas and for sure at least a CR1.

**Comment by jefrog1844** [comment]
Just ran the update and so far no issues

**Comment by gsmet** [comment]
Thanks Jeff!Message ID:
***@***.***>

>

**Comment by stephan-strate** [comment]
> It simplifies how Kotlin is handled by our classloader, which should ease the work on Kotlin-based Quarkus extensions.

Are there any kotlin-based Quarkus extensions yet? Is there a plan to have a template for it?

**Reply by maxandersen** [reply]
Not that I know of. 

Better to open separate discussion on it. Should theoretically be possible if someone is willing to try.
